test/entity.dart

```diff
@@ -11,41 +11,41 @@ class TestingUnknownAnnotation {
 class TestEntity {
   @Id()
   @TestingUnknownAnnotation()
-  int id;
+  int /*?*/ id;
```

Comment:
Should the ID property (allowed to) be nullable?

Reply:
yep, normal behaviour when inserting a new TestEntity. No need to require specifying `id=0`